static/sass/styles.scss

```diff
@@ -1,3 +1,7 @@
+// colour definitions
+$color-brand: #00302f; // TODO: this is a temprary brand colour
```

Comment:
This means that you want to wait before merging it?

Reply:
We'll be able to merge using this colour, it's just temporary until we decide to change it at a later date.

But don't merge yet as it's in design review :)

Reply:
ok let's add labels to be sure no one merges it

Reply:
It was just copied from new home page PR.
This is a brand colour that we use for now as the final snapcraft colour was not decided yet. But, as I understand we can use it until we have the final one.

Reply:
@Greggless could you clarify - is it okay to merge with this colour? Or do we need a "Don't merge" label?